browser.go

```diff
@@ -189,6 +189,14 @@ func (b *Browser) SetValue(line int) {
 	C.go_fltk_Browser_set_value((*C.Fl_Browser)(b.ptr()), C.int(line))
 }
 
+func (b *Browser) SetSelected(line int, val int) bool {
```

Comment:
It seems that val can be bool, as it can only rake two values 1 - select, 0 - unselect